scripts/gulpfiles/config.js

```diff
@@ -18,18 +18,27 @@ var path = require('path');
 //
 // - tests/scripts/compile_typings.sh
 // - tests/scripts/check_metadata.sh
-module.exports = {
-  // Directory to write compiled output to.
-  BUILD_DIR: 'build',
+// - tests/scripts/update_metadata.sh
+// - blockly_uncompressed.js (for location of deps.js)
+// - tests/playgrounds/prepare.js (for location of deps.js)
+// - tests/mocha/index.html (for location of deps.mocha.js)
 
-  // Directory in which to assemble (and from which to publish) the
-  // blockly npm package.
-  RELEASE_DIR: 'dist',
+// Directory to write compiled output to.
+exports.BUILD_DIR = 'build';
 
-  // Directory to write typings output to.
-  TYPINGS_BUILD_DIR: path.join('build', 'typings'),
+// Dependencies file (for blockly_uncompressed.js):
+exports.DEPS_FILE = path.join(exports.BUILD_DIR, 'deps.js');
 
-  // Directory where typescript compiler output can be found.
-  // Matches the value in tsconfig.json: outDir
-  TSC_OUTPUT_DIR: path.join('build', 'ts'),
-};
+// Dependencies file (for blockly_uncompressed.js):
+exports.TEST_DEPS_FILE = path.join(exports.BUILD_DIR, 'deps.mocha.js');
```

Comment:
How does outputing these to the `build` directory relate to `build:checkin`? We no longer want them to be checked in by default?

Reply:
I was thinking you were asking if we were wanting to check in `deps*.js`, which I can confirm that (IMHO) we don't, and so `checkin:built` is not relevant, since it just checks in the `*_compressed.js` files.

But now I realise you might be trying to point out that it will _also_ unintentionally copy the `deps*.js` files out of `build/` (and, umm, also all of `build/src/` too, just for bad measure).  Ooops.

Though to be sure, "hey, do you realise that `checkin:built` will copy the deps files too?" would have been helpfully blunt in this case, because I nearly overlooked this despite your efforts.  😬

Reply:
OK, fixed.